theme.json

```diff
@@ -201,30 +201,35 @@
                     "name": "Banner background dark"
                 },
                 {
-                    "slug": "header-color-background",
+                    "slug": "typography-primary-color",
+                    "color": "var(--dswp-theme-primary-blue)",
+                    "name": "Primary color"
+                }, 
+                {
+                    "slug": "background-white",
                     "color": "var(--dswp-surface-color-background-white)",
-                    "name": "Header background color"
+                    "name": "Background White" 
                 },
                 {
-                    "slug": "menu-color-background",
-                    "color": "var(--dswp-surface-color-menus-default)",
-                    "name": "Menu background color"
+                    "slug": "background-light-gray",
```

Comment:
as we discussed, using colour names like gray, white, are going to be problematic when re-using these colours or knowing where the colours are being used.  the theme json colours probably should have something like widget-background, primary-border, or something, this colour could be used for.  if we are keeping css vars they might be ok to use them in that context